lib/util/http/bitbucket.ts

```diff
@@ -19,4 +22,34 @@ export class BitbucketHttp extends Http {
     const opts = { baseUrl, ...options };
     return super.request<T>(url, opts);
   }
+
+  async getUnpaginatedJson<T>(
```

Comment:
why do we need this? we should use the paginate param instead, like we do on other http types.

Reply:
Do you have an example you can link me to.  I searched, but only found unpaginated implementations within specific files like `lib/util/http/github.ts`

Reply:
A bit more detective work 🕵️ and found four references in
* `lib/util/http/gitea` - `GiteaHttpOptions`
* `lib/util/http/github`- `GithubHttpOptions`
* `lib/util/http/gitlab` - `GitlabHttpOptions`
* `lib/util/http/types` - `GraphqlOptions`


I'll follow the pattern and add similar to `lib/util/http/bitbucket` for `BitbucketHttpOptions`

Reply:
https://github.com/renovatebot/renovate/pull/22135